### PR TITLE
Update windows test script

### PR DESCRIPTION
**Pull request by jingxu97**
fix issues in init-buildx.sh

**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: init-buildx.sh [hunk 1]
```diff
@@ -14,10 +14,16 @@
...
+set -o nounset
+set -o pipefail
 
 export DOCKER_CLI_EXPERIMENTAL=enabled
-
+echo "set docker cli experimental"
 # Ensure we use a builder that can leverage it (the default on linux will not)
 docker buildx rm windows-builder || true
+
+echo "buildx"
```
Comment (msau42) [inline init-buildx.sh:26]: is this just for debugging or do you want to merge this? If you want to merge this, can you make the statements more specific?

Comment (jingxu97) [reply init-buildx.sh:25]: this just for debugging. Some issue with set pipefail

Comment (jingxu97) [reply init-buildx.sh]: I have to remove pipefail, it always return error: invalid option.

**Comment by jingxu97** [comment]
cc @msau42 PTAL

**Pull request merged**